cli/pcluster/utils.py

```diff
@@ -269,8 +269,9 @@ def get_instance_vcpus(region, instance_type):
     or the pricing file cannot be retrieved/parsed
```

Comment:
Comment out-to-date.

Then I'd put this important change in the commit message and in the PR description.

Reply:
Done